test/t/test_make.py

```diff
@@ -45,3 +47,32 @@ def test_8(self, bash, completion):
     @pytest.mark.complete("make -", require_cmd=True)
     def test_9(self, completion):
         assert completion
+
+
+@pytest.mark.bashcomp(require_cmd=True, temp_cwd=True)
+class TestMake2:
+    def test_github_issue_544_1(self, bash):
+        print(os.getcwd())
+        print(bash.cwd)
+        with open("Makefile", "w") as f:
```

Comment:
It seems this could be placed in a static fixture dir of its own instead of generating on the fly in a temp cwd.

Reply:
Do you mean we add a Makefile at `/test/fixture/make/somewhere` in the tree?

Reply:
Yep, that's what I was thinking of. I think it's better to have static files there as is, and generate only when that's not feasible (e.g. dynamic content, chars that are problematic in some environments in file names).

Reply:
OK

Reply:
I have added a static file `/test/fixtures/make/test2/Makefile` in the tree.